CHANGES_NEXT_RELEASE

```diff
@@ -1,4 +1,5 @@
 - Fix: deprecated mongo warnings about save and remove methods (#806)
 - Fix: remove unnecessary env vars (PERSEO_MONGO_HOST and PERSEO_CORE_URL) from Dockerfile (#810)
+- Upgrade jinja2 from 3.1.4 to 3.1.5 due to CVE-2024-56201 and CVE-2024-56326
```

Comment:
```suggestion
```

The fix is not actually related with the component itself, but with the RTD framework.